graphrag/index/workflows/default_workflows.py

```diff
@@ -91,3 +129,18 @@
     create_final_communities: build_create_final_communities_steps,
     generate_text_embeddings: build_generate_text_embeddings_steps,
 }
+
+basic_workflows: dict[str, Any] = {
```

Comment:
Could we type the value of the dict as `Callable[[GraphRagConfig, PipelineRunContext, VerbCallbacks], Awaitable[None]]` instead of `Any` since they all have the same signature?